src/main/java/com/pusher/client/PusherOptions.java

```diff
@@ -23,6 +24,7 @@
 
     private static final long DEFAULT_ACTIVITY_TIMEOUT = 120000;
     private static final long DEFAULT_PONG_TIMEOUT = 30000;
+    private Proxy proxy = null;
```

Comment:
Group this with the other members, not the constants please.

There's no need to provide an explicit `null` assignment here, member variables are always initialised (see the authorizer). If you don't like that then explicit assignment is fine, but should be consistent with the rest of the class.